CHANGELOG.md

```diff
@@ -1,27 +1,33 @@
-# 1.3.2
+# Change Log
 
-* Fix: promisify all Docker functions
+All notable changes to this project will be documented in this file
+automatically by Versionist. DO NOT EDIT THIS FILE MANUALLY!
+This project adheres to [Semantic Versioning](http://semver.org/).
 
-# 1.3.1
+# 1.3.2 - 2016-11-18
 
-* Fixed promisifying docker.runAsync so it returns all callback results.
+* Fix: promisify all Docker functions [Pablo Carranza Velez]
 
-# 1.3.0
+# 1.3.1 - 20160-11-09
```

Comment:
It was actually a version from the future, but thanks for reminding me we have to keep that kind of things secret.